test/journeys/specs/recents/dataApi/space-type-filter.js

```diff
@@ -0,0 +1,186 @@
+import {assert} from 'chai';
+
+import {createSpace, disconnectDevices, registerDevices, setupGroupTestUsers} from '../../../lib/test-users';
+
+import {jobNames, renameJob, updateJobStatus} from '../../../lib/test-helpers';
+import {elements} from '../../../lib/test-helpers/recents-widget';
+
+describe('Widget Recents with Space Type Filter: Data API', () => {
```

Comment:
Can we combine all of the filter tests into one suite? No need to setup all the test users and spaces again. Need this for smoke tests and global as well.

Reply:
+1 on this, would save a lot of time in the tests